docs/deploy/installation.md

```diff
@@ -44,9 +44,15 @@ The reference IAM policies contain the following permissive configuration:
     "Effect": "Allow",
     "Action": [
         "ec2:AuthorizeSecurityGroupIngress",
-        "ec2:RevokeSecurityGroupIngress"
+        "ec2:RevokeSecurityGroupIngress",
+        "ec2:DeleteSecurityGroup"
     ],
     "Resource": "*"
+    "Condition": {
+        "Null": {
+            "aws:ResourceTag/elbv2.k8s.aws/cluster": "false"
```

Comment:
this won't work since the instance/ENI SG might not have the tag you've specified here